__tests__/app.test.js

```diff
@@ -30,21 +29,73 @@ describe('* invalid endpoints', () => {
         .get('/doesn"t/exist')
         .expect(404)
         .then((res) => {
-            console.log(res.body.msg)
             expect(res.body.msg).toEqual('Endpoint not found')
-
+            
         })
     })
-
+    
     it('returns a 404 status code when making an invalid put request', () => {
         return request(app)
         .get('/doesn"t/exist')
         .send({msg: 'This should message should never '})
-        .expect(404)
         .then((res) => {
-            console.log(res.body.msg)
             expect(res.body.msg).toEqual('Endpoint not found')
+            
+        })
+    })
+})
+
+describe('GET /api', () => {
+    it('returns a 200 status code and returns an object of available end points', () => {
+        return request(app)
+        .get('/api')
+        .expect(200)
+        .then((res) => {
+            const endPoints = res.body.newEndPoints
+            expect(res.headers['content-type']).toMatch(/application\/json/)
+            expect(endPoints).toHaveProperty('GET /api')
+            expect(endPoints).toHaveProperty('GET /api/topics')
+            expect(endPoints).toHaveProperty('GET /api/articles')
 
         })
     })
 })
+
+describe.only('GET /api/articles/:article_id', () => {
+    it('returns a 200 status code when valid id given and returns an object with correct properties', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then((res) => {
+            const article = res.body.article[0]
```

Comment:
As it requests for  a single article, it'd be better if only the article comes back (as an object instead of an array)